data/ytt/docs/ytt-latest-toc.yml

```diff
@@ -17,6 +17,10 @@ toc:
         url: /data-values-vs-overlays
       - page: Injecting secrets
         url: /injecting-secrets
+  - title: Concepts
```

Comment:
should this be in the intro section next to "how it works"?

Reply:
That is _exactly_ where I put it, at first.

However, I realized that one does not _need_ to know this stuff to get started with `ytt`. It's far more useful for someone who has been just slightly confused while annotating (especially overlays).

The difference between _this_ page and "How it Works" is that one can walk in with an intuitive understanding of YAML, but they currently have to discover the `ytt` pipeline. In fact, I suspect most new users to `ytt` will only look at that picture at the top, note the four black boxes, and move on... when they start needing to wonder about the order of things, they'll come back to the "How it Works" and go deeper with specific questions.

I also want to keep "How it Works" prominent — it's the fundamental mental model on which the rest of the reference material hangs.

We have plenty of other "Concepts" that can flesh out this section, in the future.... I'm just scratching a major itch.

Reply:
ok then lets move it before Techniques? it feels it's a bit too late in the sidebar.